<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LSnapDrawPanel</class>
 <widget class="QFrame" name="LSnapDrawPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Draw Panel</string>
  </property>
  <property name="styleSheet">
   <string>QFrame { background-color: rgba(50,50,50,220); border-radius: 4px; }</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QPushButton" name="lineWidthBtn">
     <property name="text">
      <string>线宽: 2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="rectBtn">
     <property name="text">
      <string>矩形</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="ellipseBtn">
     <property name="text">
      <string>椭圆</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
